### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by Farida-Hamid**
> In this project, I built a simple HTML list of To-Do tasks. The list is styled according to the specifications listed in the project description. This simple web page is built using Webpack and served by a Webpack dev server.

### CODE REVIEW

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @Farida-Hamid,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 

- [x] Kindly, make sure you add instruction in your `README` file `Getting Started` section how to run the project. And make sure your project is rendering the list of items you have in the page correctly. Here is what currently shown in the page. 👇 

image


## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 1]
```diff
@@ -1,39 +1,33 @@
...
+  list.forEach((task) => {
+    listContainer.innerHTML += `
+    <li>
+      <input type="checkbox">
+      <p>${task.description}</p>
+    </li>`;
+  });
+};
+
+loader();
```
Comment (DeliceLydia) [inline src/index.js:33]: - [ ] Please implement event listener on the checkbox (change).
- [ ] The task object's values for `completed` should be updated upon user actions.
- [ ] Please work on function for the "Clear all completed" button using the `Array.filter()`

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Farida-Hamid, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by elyor-doniyorov — APPROVED** [review]
Hi @Farida-Hamid, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Descriptive Readme file
✔️ Webpack is used to set up the project 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**